include/deal.II/dofs/dof_handler.h

```diff
@@ -535,8 +538,8 @@ class DoFHandler : public Subscriptor
    * Invalid active FE index which will be used as a default value to determine
    * whether a future FE index has been set or not.
    */
-  static const active_fe_index_type invalid_active_fe_index =
-    static_cast<active_fe_index_type>(-1);
+  static const types::fe_index invalid_active_fe_index =
+    static_cast<types::fe_index>(-1);
```

Comment:
Why not move this to numbers?

Reply:
I will do this in a follow-up PR. I'll make small changes, that way we can easily identify if some commit breaks something.